<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Router Keygen</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_private_key">
      <property name="text">
       <string>Private key:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPlainTextEdit" name="edit_private_key">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="button_copy_private_key">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources/keygen.qrc">
            <normaloff>:/img/clipboard-arrow.png</normaloff>:/img/clipboard-arrow.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_save_private_key">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources/keygen.qrc">
            <normaloff>:/img/disk.png</normaloff>:/img/disk.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_public_key">
      <property name="text">
       <string>Public key:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="edit_public_key">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="button_copy_public_key">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources/keygen.qrc">
            <normaloff>:/img/clipboard-arrow.png</normaloff>:/img/clipboard-arrow.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_save_public_key">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources/keygen.qrc">
            <normaloff>:/img/disk.png</normaloff>:/img/disk.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="button_generate">
      <property name="text">
       <string>Generate</string>
      </property>
      <property name="icon">
       <iconset resource="resources/keygen.qrc">
        <normaloff>:/img/key.png</normaloff>:/img/key.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources/keygen.qrc"/>
 </resources>
 <connections/>
</ui>
